[
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.hl6HsBTsCq"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:58:47 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:32 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\\fatt_24_update_tax_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>\n<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\\fatt_24_update_tax_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#7 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#8 {main}</pre>\n for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(27): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(311): Fattura24\\fatt_24_add_column_to_tax_table('used_for_shippi...', 'INT', 1, 0)\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_add_column_to_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, activation, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(312): Fattura24\\fatt_24_insert_installation_log('activation')\n#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#9 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#10 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated billing cb postmeta, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated billing...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated old wc orders postmeta, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated old wc ...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated order creation options, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated order c...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fattura24 tax settings, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated fattura...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/src/uty.php(80): Fattura24\\fatt_24_update_tax_table()\n#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#9 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#10 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>\n<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(80): Fattura24\\fatt_24_update_tax_table()\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fattura24 tax table, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated fattura...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated invoice creation options, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated invoice...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:39 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-07 12:59:39')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fe issue number default option, 2025-05-07 12:59:39</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated fe issu...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-07 12:59:39') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:14 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\\fatt_24_update_tax_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>\n<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\\fatt_24_update_tax_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#7 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#8 {main}</pre>\n for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(27): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(311): Fattura24\\fatt_24_add_column_to_tax_table('used_for_shippi...', 'INT', 1, 0)\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_add_column_to_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-07 13:00:18')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, activation, 2025-05-07 13:00:18</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(312): Fattura24\\fatt_24_insert_installation_log('activation')\n#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#9 /internal/eval.php(20): activate_plugin('fattur\n",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.Kxez9cgdNp"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:58:47 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:32 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\\fatt_24_update_tax_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>\n<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\\fatt_24_update_tax_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#7 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#8 {main}</pre>\n for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(27): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(311): Fattura24\\fatt_24_add_column_to_tax_table('used_for_shippi...', 'INT', 1, 0)\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_add_column_to_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, activation, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(312): Fattura24\\fatt_24_insert_installation_log('activation')\n#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#9 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#10 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated billing cb postmeta, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated billing...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated old wc orders postmeta, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated old wc ...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated order creation options, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated order c...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fattura24 tax settings, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated fattura...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/src/uty.php(80): Fattura24\\fatt_24_update_tax_table()\n#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#9 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#10 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>\n<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(80): Fattura24\\fatt_24_update_tax_table()\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fattura24 tax table, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated fattura...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated invoice creation options, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated invoice...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:39 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-07 12:59:39')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fe issue number default option, 2025-05-07 12:59:39</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated fe issu...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-07 12:59:39') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:14 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\\fatt_24_update_tax_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>\n<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\\fatt_24_update_tax_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#7 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#8 {main}</pre>\n for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(27): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(311): Fattura24\\fatt_24_add_column_to_tax_table('used_for_shippi...', 'INT', 1, 0)\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_add_column_to_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-07 13:00:18')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, activation, 2025-05-07 13:00:18</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(312): Fattura24\\fatt_24_insert_installation_log('activation')\n#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#9 /internal/eval.php(20): activate_plugin('fattur\n",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.oRk7duqdzR"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:58:47 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:32 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\\fatt_24_update_tax_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>\n<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\\fatt_24_update_tax_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#7 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#8 {main}</pre>\n for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(27): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(311): Fattura24\\fatt_24_add_column_to_tax_table('used_for_shippi...', 'INT', 1, 0)\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_add_column_to_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, activation, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(312): Fattura24\\fatt_24_insert_installation_log('activation')\n#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#9 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#10 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated billing cb postmeta, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated billing...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated old wc orders postmeta, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated old wc ...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated order creation options, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated order c...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fattura24 tax settings, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated fattura...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/src/uty.php(80): Fattura24\\fatt_24_update_tax_table()\n#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#9 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#10 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>\n<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(80): Fattura24\\fatt_24_update_tax_table()\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fattura24 tax table, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated fattura...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:38 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-07 12:59:38')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated invoice creation options, 2025-05-07 12:59:38</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated invoice...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-07 12:59:38') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 10:59:39 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-07 12:59:39')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fe issue number default option, 2025-05-07 12:59:39</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated fe issu...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-07 12:59:39') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "level": "NOTICE",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:14 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href=\"https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/\">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\\fatt_24_update_tax_table()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>\n<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\\fatt_24_update_tax_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#7 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#8 {main}</pre>\n for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(27): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(311): Fattura24\\fatt_24_add_column_to_tax_table('used_for_shippi...', 'INT', 1, 0)\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_add_column_to_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-07 13:00:18')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, activation, 2025-05-07 13:00:18</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(312): Fattura24\\fatt_24_insert_installation_log('activation')\n#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#9 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#10 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-07 13:00:18') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-07 13:00:18')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated billing cb postmeta, 2025-05-07 13:00:18</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated billing...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-07 13:00:18') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-07 13:00:18')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated old wc orders postmeta, 2025-05-07 13:00:18</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated old wc ...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-07 13:00:18') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-07 13:00:18')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated order creation options, 2025-05-07 13:00:18</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated order c...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-07 13:00:18') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-07 13:00:18')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fattura24 tax settings, 2025-05-07 13:00:18</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated fattura...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-07 13:00:18') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "Unknown query type: DESC.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC wp_fattura_tax</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)\n#3 /wordpress/wp-content/plugins/fattura24/src/uty.php(80): Fattura24\\fatt_24_update_tax_table()\n#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#9 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#10 {main}</pre>\n for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>\n<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(80): Fattura24\\fatt_24_update_tax_table()\n#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#9 {main}</pre>\n for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-07 13:00:18')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fattura24 tax table, 2025-05-07 13:00:18</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated fattura...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-07 13:00:18') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:18 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-07 13:00:18')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated invoice creation options, 2025-05-07 13:00:18</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated invoice...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-07 13:00:18') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 20 datatype mismatch.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "gpp-shortcodes",
    "details": "[07-May-2025 11:00:19 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-07 13:00:19')</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fe issue number default option, 2025-05-07 13:00:19</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')\n#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')\n#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)\n#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\\fatt_24_insert_installation_log('updated fe issu...')\n#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\\fatt_24_update_actions()\n#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\\{closure}(false)\n#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)\n#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')\n#11 {main}</pre>\n for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-07 13:00:19') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\\{closure}, Fattura24\\fatt_24_update_actions, Fattura24\\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"gpp-shortcodes\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/gpp-shortcodes.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/gpp-shortcodes.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/gpp-shortcodes.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/gpp-shortcodes.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n",
    "log": "/logs/plugins/g/gpp-shortcodes/error.json"
  }
]
